ec/src/models/short_weierstrass/affine.rs

```diff
@@ -125,7 +124,7 @@ impl<P: SWCurveConfig> Affine<P> {
     ///
     /// The results are sorted by lexicographical order.
     /// This means that, if `P::BaseField: PrimeField`, the results are sorted as integers.
-    fn get_ys_from_x_unchecked(x: P::BaseField) -> Option<(P::BaseField, P::BaseField)> {
+    pub fn get_ys_from_x_unchecked(x: P::BaseField) -> Option<(P::BaseField, P::BaseField)> {
```

Comment:
Deserialize uses `get_ys_from_x_unchecked`. Should I move `get_ys_from_x_unchecked` and `get_point_from_x_unchecked` to the curve config too? If not, I need to make this public.

Reply:
You can make it `pub`